Mathlib/RingTheory/Binomial.lean

```diff
@@ -450,6 +452,54 @@ theorem choose_add_smul_choose [NatPowAssoc R] (r : R) (n k : ℕ) :
   rw [choose_smul_choose (r + k) (n + k) k (Nat.le_add_left k n), Nat.add_sub_cancel,
     add_sub_cancel_right]
 
+end
+
+/-- Pochhammer version of Chu-Vandermonde identity -/
+theorem descPochhammer_smeval_add [Ring R] {r s : R} (k : ℕ) (h: Commute r s) :
+    (descPochhammer ℤ k).smeval (r + s) = ∑ ij ∈ Finset.HasAntidiagonal.antidiagonal k,
+    Nat.choose k ij.1 * ((descPochhammer ℤ ij.1).smeval r * (descPochhammer ℤ ij.2).smeval s) := by
+  induction k with
+  | zero => simp [descPochhammer_zero, Finset.Nat.antidiagonal_zero, Finset.sum_singleton]
```

Comment:
```suggestion
  | zero => simp
```
These are all already `@[simp]` lemmas, so I'm not sure it's helpful to add the explicitly unless you are using `simp only`.